example/09_convnd_fwd/convnd_fwd_xdl_fp8.cpp

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
Could You fix each Copyright header? For new files just 2024, for updated -2024.